src/components/molecules/LoadingPage/LoadingPage.tsx

```diff
@@ -27,30 +36,35 @@ const quotes = IS_BURN
     ]
   : ["Loading..."];
 
-export const LoadingPage = () => {
+const _LoadingPage = () => {
   const [quote, setQuote] = useState("Loading...");
 
   useInterval(
-    () => {
+    useCallback(() => {
       setQuote(quotes[Math.floor(Math.random() * quotes.length)]);
-    },
+    }, []),
     IS_BURN ? 1000 : undefined
   );
 
+  const { isShown: animate, show: startAnimation } = useShowHide();
+  //this prevents "flickering" of the animation on page load
```

Comment:
This sounds like a hacky solution. Can we think of something else?

Reply:
So the problem is that on initial page load (for example when accessing this new route `/loading`) the LoadingPage component not just re-renders multiple times but it 
- renders
- then in some time is removed from the render tree, unrenders
- then it's rendered again. 

Which means that animation is started, then when the component is removed and quickly brought back (it is just visible in the debug/logs but not to the naked eye) animation is restarted which results in some sort of _flashing_ of the diamond logo. Then it proceeds as normal.

I thought that this initial time is important for the overall user experience of this loading page but didn't found a better solution to just delay the start of the animation.

Reply:
Why is this not an issue with our current implementation? There should be a way to fix this. 300 delay is a lot and it just hides/skips the problem.

Reply:
It's actually an issue:) The current animation flickers too but I think you just don't notice it. On my machine, I barely even notice the loading page at all.
![flickering](https://user-images.githubusercontent.com/16973041/130205991-de58a81c-ed5c-464e-92f2-9b8bfeacaa9f.gif)

Reply:
The other solution is to make the `LoadingPage` render all the time. We can rename it to a `LoadingScreen` or similar, move to the top of the component tree, create a context, wrap the whole component tree in it and then make a hook e.g. `useLoading` which will accept a boolean to hide or show this loading screen. Then it should work.

Reply:
I think you should show it so Sofi and let her decide, IMO we should leave it as it is and let it flicker.

Reply:
Okay, I removed the delay in https://github.com/sparkletown/sparkle/pull/2005/commits/44c45c4b89f7f394f70398fbf980eb9330d106ec so it flickers now, but as the default animation in staging does.

I'll contact @sofisparkle and consult whether this flicker needs improvements.